### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by YouSafe**
I updated the dependencies and tested the examples

### CODE REVIEW

**Comment by YouSafe** [comment]
Thank you to @stefnotch for finding the high DPI bug.

**Comment by MatchaChoco010** [comment]
Sorry for the delay in noticing! I will check when I get home from work today.

File: src/integration.rs [hunk 1]
```diff
@@ -70,13 +63,13 @@ impl<A: AllocatorTrait> Integration<A> {
...
     ) -> Self {
-
         // Create context
         let context = Context::default();
         context.set_fonts(font_definitions);
         context.set_style(style);
 
-        let egui_winit = egui_winit::State::new(&event_loop);
+        let mut egui_winit = egui_winit::State::new(&event_loop);
+        egui_winit.set_pixels_per_point(scale_factor as f32);
```
Comment (stefnotch) [inline src/integration.rs:72]: ^ This line was missing. Previously, on high DPI monitors, the egui-winit-ash-integration would stretch the textures too much.

**Comment by MatchaChoco010** [comment]
I have verified that both examples work on my machine.
Also, the code looks fine, so I'll merge them.

**Comment by MatchaChoco010** [comment]
I just published 0.4.1! Thanks for your contribution!
https://crates.io/crates/egui-winit-ash-integration/0.4.1

**Pull request merged**